jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -13,38 +13,38 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.35",
-    "@fortawesome/free-solid-svg-icons": "^5.15.3",
-    "@fortawesome/vue-fontawesome": "^2.0.2",
-    "@triply/yasqe": "^4.2.7",
-    "@triply/yasr": "^4.2.8",
-    "axios": "^0.21.1",
-    "bootstrap": "^4.6.0",
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
+    "@fortawesome/vue-fontawesome": "^2.0.6",
+    "@triply/yasqe": "^4.2.20",
+    "@triply/yasr": "^4.2.20",
+    "axios": "^0.24.0",
+    "bootstrap": "^5.1.3",
```

Comment:
Sorry, when I finished the PR they were finishing the version 5. I upgraded Skosmos from 3 to 5, but forgot to upgrade the Fuseki PR. Will try to review what changed, and what needs to be fixed (luckily we don't use Bootstrap per se, but Bootstrap Vue, which abstracts the Bootstrap syntax for us)